oeps/architectural-decisions/oep-0041-arch-async-server-event-messaging.rst

```diff
@@ -475,10 +475,16 @@ Having a clearly defined set of events would allow for simpler third party
 integration in areas like student learner event processing and course catalog
 management.
 
+------------
+
+.. note:: All future decisions related to :doc:`Open edX Events <openedx-events:index>` that adhere to these guidelines for asynchronous event messaging will now be documented and made publicly available through the Architectural Decision Records (ADRs) in :doc:`Open edX Events ADRs <openedx-events:decisions/index>`. This includes decisions regarding the Event Bus, schema format, serialization, evolution, and design practices for event-driven architecture.
+
 **************
 Change History
 **************
 
+2024-01-14: Added note about decisions related to Open edX Events being documented and made publicly available through the Architectural Decision Records (ADRs) in the repository.
```

Comment:
There may be more places that warrant this change, but “the repository” is vague and by default might mean “this repository”, but it doesn’t.
```suggestion
2024-01-14: Added note about decisions related to Open edX Events being documented and made publicly available through the Architectural Decision Records (ADRs) in the openedx-events repository.
```

Reply:
This comment still applies here and below. Otherwise, I think this is good to go. Thank you.

Reply:
Thank you. I added your suggestions here: https://github.com/openedx/open-edx-proposals/pull/662/commits/b2dce77c79c76abdffd6fe01f721f973693c70e8